tests/test_formatter.py

```diff
@@ -137,7 +136,7 @@ def test_numerical_hello_world_domain_formatter():
             "    (:action say-hello-world",
             "        :parameters (?neighbor)",
             "        :precondition (<= (hello_counter ?neighbor) 3)",
-            "        :effect (and (increase (hello_counter ?neighbor) 1))",
+            "        :effect (increase (hello_counter ?neighbor) 1)",
```

Comment:
We ok with this? Comes from the fact that `And` has `idempotency = True`

Reply:
I'm ok with it.